Technology/Artificial Intelligence/interactive & great resources/README.md

```diff
@@ -0,0 +1,45 @@
+# Interactive Tools and other great Resources
+
+# Interactive Tools
+
+## Distill: Exploring Neural Networks with Activation Atlases
+
+Feature inversion to visualize millions of activations from an image classification network leads to an explorable activation atlas of features the network has learned. This can reveal how the network typically represents some concepts.
+
+[<p align="center"><img src="https://github.com/ash11sh/co-learning-lounge/raw/master/imagess/activation_atlas.png" width="1000"></p>](https://distill.pub/2019/activation-atlas/)
+
+
+## A visual introduction to Machine Learning
+Available in many different languages.
+
+[<p align="center"><img src="https://github.com/ash11sh/co-learning-lounge/raw/master/imagess/intro_ML.png" width="1000"></p>](http://www.r2d3.us/visual-intro-to-machine-learning-part-1/)
+
+
+## Interactive Deep Learning Playground
+New to Deep Learning? Tinker with a Neural Network in your browser.
+
+[<p align="center"><img src="https://github.com/ash11sh/co-learning-lounge/raw/master/imagess/dl_playground.png" width="1000"></p>](https://playground.tensorflow.org)
+
+## Initializing neural networks
+
+Initialization can have a significant impact on convergence in training deep neural networks. Simple initialization schemes can accelerate training, but they require care to avoid common pitfalls. In this post, deeplearning.ai folks explain how to initialize neural network parameters effectively.
+
+[<p align="center"><img src="https://github.com/ash11sh/co-learning-lounge/raw/master/imagess/weight_init.png" width="1000"></p>](https://www.deeplearning.ai/ai-notes/initialization/)
+
+# Math
```

Comment:
Will have "Maths" section under AT folder. Request you to keep in its README file.